testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -1885,6 +1886,36 @@ QUnit.test("Adaptivity should be available for horizontal orientation only", fun
     assert.equal($adaptiveContainer.length, 0, "adaptiveContainer was not rendered");
 });
 
+QUnit.test("maxHeight should be 90% of maximum of top or bottom offsets when height of overlay content more windows height", function(assert) {
+    var getData = function(count) {
+        var result = [];
+        for(var i = 0; i < count; i++) {
+            result.push({ text: "item_" + i });
+        }
+        return result;
+    };
+
+    new Menu(this.$element, {
+        items: getData(10),
```

Comment:
You can mock outerHeight function in the menu object and replace 10 items with only one item to make this test shorter